desktop/cmp/button/ButtonGroup.ts

```diff
@@ -33,6 +34,8 @@ export interface ButtonGroupProps<M extends HoistModel = null>
 
     /** True to render in a vertical orientation. */
     vertical?: boolean;
+
+    ref?: ForwardedRef<any>;
```

Comment:
Looking at the underlying blueprint component, I think this should be `ForwardedRef<HTMLDivElement>`?

Reply:
Fixed.